README.md

```diff
@@ -1,2 +1,49 @@
-# dp-import-cantabular-dimension-options
 dp-import-cantabular-dimension-options
+================
+dp-import-cantabular-dimension-options
+
+### Getting started
+
+* Run `make debug`
+
+The service runs in the background consuming messages from Kafka.
+An example event can be created using the helper script, `make produce`.
+
+### Dependencies
+
+* Requires running…
+  * [kafka](https://github.com/ONSdigital/dp/blob/main/guides/INSTALLING.md#prerequisites)
+* No further dependencies other than those defined in `go.mod`
+
+### Configuration
+
+| Environment variable         | Default                           | Description
+| ---------------------------- | --------------------------------- | -----------
+| BIND_ADDR                    | localhost:26300                    | The host and port to bind to
```

Comment:
As the service is just replaced with the event type, I think we don't need to change the port (unless there is some other reason to change it?) I would suggest keeping port `26200`